### DISCUSSION TITLE
2023-07-31 정기 백엔드 회의

### DISCUSSION THREAD

**Discussion by scarf005** [jiphyeonjeon-42/backend#613] [General]
[지난주 회의](https://www.collabee.co/project/163329/issue/postDetail/2392904)

## 일정

14:10/14:40 - 지난 주 계획 달성 분석 및 개선 방안 논의
15:00/16:00 - 궁금한 점 물어보기, 칸반 보드 작성하기, asignee 정하기

### 코드 같이 읽어보기

- https://github.com/jiphyeonjeon-42/backend/pull/599

### @scarf005 - 문서화 미비에 대한 반성

- https://github.com/jiphyeonjeon-42/backend/issues/611

### 프로젝트 보드에 개발 일정 구체적으로 적기

[프로젝트 보드](https://github.com/orgs/jiphyeonjeon-42/projects/8/views/2?query=is%3Aopen+sort%3Aupdated-desc)

**Comment by nyj001012** [comment]
궁금한 점
- 스키마와 타입의 차이: 어떤 부분에는 스키마를 정의해서 사용하고, 어떤 곳에는 타입을 정의해서 사용하던데 적절한 구분 방법이 궁금합니다.

**Reply by scarf005** [reply]
혹시 연관 코드 링크를 공유해주실수 있을까요?

**Reply by nyj001012** [reply]
type: https://github.com/jiphyeonjeon-42/backend/blob/e2fbd64c60e10281c4e0691b48319778e6b1e6f9/backend/src/v2/reviews/service/service.ts#L15C1-L17C20

schema: https://github.com/jiphyeonjeon-42/backend/blob/e2fbd64c60e10281c4e0691b48319778e6b1e6f9/backend/src/v2/shared/user.ts#L15-L17

여기요 :)

**Comment by Yonaim** [comment]
오늘 나온 얘기...
- /backend/backend와 같은 경로가 말이되는가 디렉토리 이름 바꾸자 <- 리팩토링 작업 전부 끝나고 나서 고려

**Comment by scarf005** [comment]
지난주에 콜라비에 할 작업들을 적었는데 잘 진행되지 않아서 이번주에는 잘 진행하려면 어떤 점을 개선하면 좋을까요

**Reply by scarf005** [reply]
- 구체적인 이슈, 데드라인이 없어 무엇을 해야 할지 어려움

**Reply by nyj001012** [reply]
방목형 일정관리

**Reply by Yonaim** [reply]
지금 프로젝트 탭 내에 TODO가 있긴한데 '리팩토링' 하나로 퉁쳐버리는 바람에 진행 상황을 공유하는데 어려움이 있다고 생각합니다. 예나님과 저의 팀을 제외한 나머지 팀의 상황을 전혀 모름...

**Reply by scarf005** [reply]
프로젝트 roadmap 주도 개발로 해결하기로 결정!

![image](https://github.com/jiphyeonjeon-42/backend/assets/54838975/2f879612-fdc6-455d-a665-c8c298105863)

https://github.com/orgs/jiphyeonjeon-42/projects/8/views/3?query=is%3Aopen+sort%3Aupdated-desc

**Comment by scarf005** [comment]
가이드 작성하기
우산 이슈 만들기 이슈폼 작성하기
프로젝트 보드 활용법 작성하기
코드리뷰하기

프로젝트 보드에 지난주 하기로한거 추가하기

No tracked by를 전부 우산씌워주기

**Comment by scarf005** [comment]
아이디어 아무말 대잔치

book bookinfo 헷갈리니까 v2에서는 명칭 바꾸기
kysely 쓰기
deno 쓰기
코딩 컨벤션 바꾸기

**Comment by Yonaim** [comment]
- 멤버로 추가되어있지 않은 팀원 분들은 assign이 불가함. 멤버에 추가하고 각자 할당된 이슈에 assign하기

**Reply by scarf005** [reply]
![image](https://github.com/jiphyeonjeon-42/backend/assets/54838975/4a9ba506-6bb5-4285-a51e-aeb2adfde5d6)

멤버 및 팀 추가 완료! `@백엔드` 로 핑 가능합니다.

**Comment by Yonaim** [comment]
@scarf005 이것은 테스트 핑입니다
ㅎㅎ

**Reply by scarf005** [reply]
굳
